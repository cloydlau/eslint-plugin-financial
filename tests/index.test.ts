// @ts-ignore
import financial from '../src/index'

it('entry', () => {
  expect(financial).toMatchSnapshot()
})
