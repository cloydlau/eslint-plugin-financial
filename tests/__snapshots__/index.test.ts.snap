// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entry 1`] = `
Object {
  "configs": Object {
    "recommended": Object {
      "plugins": Array [
        "financial",
      ],
      "rules": Object {
        "financial/no-division": "error",
        "financial/no-float-calculation": "error",
      },
    },
  },
  "rules": Object {
    "no-division": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "category": "Possible Errors",
          "description": "disallow division",
          "recommended": true,
          "url": "https://github.com/ulivz/eslint-plugin-financial/blob/master/docs/rules/no-division.md",
        },
        "type": "problem",
      },
    },
    "no-float-calculation": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "category": "Possible Errors",
          "description": "disallow floating point calculation",
          "recommended": true,
          "url": "https://github.com/ulivz/eslint-plugin-financial/blob/master/docs/rules/no-floating-point-calculation.md",
        },
        "type": "problem",
      },
    },
  },
}
`;
